### PR TITLE
Cutscene macro calls

### PR DESCRIPTION
**Pull request by LuisBoto**
Added several project-wide macro calls, mainly on cutscene files

### CODE REVIEW

File: data/dungeon/itempat.inc [hunk 1]
```diff
@@ -32,7 +32,9 @@ itempat:
 .byte 0xff, 0xff, 0x17, 0x00, 0x00, 0x71, 0x77, 0xff, 0x00, 0x51, 0x55, 0x6f, 0x00, 0xf1, 0x77, 0xf5, 0x00, 0x71, 0x66, 0x56, 0x00, 0x51, 0x66, 0x66, 0x00, 0x10, 0x11, 0x11, 0x00, 0x00, 0x00, 0x00
 .byte 0x00, 0x00, 0x00, 0x00, 0x77, 0x77, 0x16, 0x00, 0x65, 0x51, 0x15, 0x00, 0x17, 0x77, 0x17, 0x00, 0x65, 0x66, 0x16, 0x00, 0x66, 0x66, 0x15, 0x00, 0x11, 0x11, 0x01, 0x00, 0x00, 0x00, 0x00, 0x00
 .byte 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x10, 0x11, 0x00, 0x10, 0x91, 0xa9
-.byte 0x00, 0x91, 0xaa, 0x88, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x11, 0x01, 0x00, 0x00, 0x9a, 0x19, 0x01, 0x00
+.byte 0x00
+rotate 0xaa, 0x88, south
```
Comment (SethBarberee) [inline data/dungeon/itempat.inc:36]: I'm not sure that script commands are used here.. this is item palette stuff.

File: data/dungeon/etcfont.inc [hunk 2]
```diff
@@ -3,7 +3,9 @@ etcfont:
 .byte 0x32, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x10, 0x11, 0x11, 0x01
 .byte 0x11, 0x11, 0x11, 0x11, 0x11, 0x11, 0x11, 0x11, 0x10, 0x11, 0x11, 0x01, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x10, 0x11, 0x00, 0x11, 0x11, 0x11
 .byte 0x10, 0x11, 0x11, 0x11, 0x10, 0x11, 0x11, 0x11, 0x00, 0x11, 0x11, 0x11, 0x00, 0x00, 0x10, 0x11, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x11, 0x01, 0x00, 0x00, 0x11, 0x11, 0x11, 0x00
-.byte 0x11, 0x11, 0x11, 0x01, 0x11, 0x11, 0x11, 0x01, 0x11, 0x11, 0x11, 0x00, 0x11, 0x01, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x11
+.byte 0x11, 0x11, 0x11, 0x01, 0x11, 0x11, 0x11, 0x01, 0x11, 0x11, 0x11, 0x00, 0x11
+warpLocation 0
```
Comment (SethBarberee) [inline data/dungeon/etcfont.inc:7]: This is all graphics for fonts. This shouldn't have any scripting commands.

File: data/system_sbin.s [hunk 3]
```diff
@@ -2818,7 +2819,9 @@ gUnknown_830F904:
 .global gUnknown_830F94C
 gUnknown_830F94C:
 .byte 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0xf0, 0xf0, 0x00, 0x00, 0x00, 0x00, 0x00, 0x0f, 0x00, 0x00, 0x00, 0x00, 0xff, 0xff
-.byte 0x0f, 0x00, 0x00, 0x00, 0x00, 0x0f, 0x00, 0x00, 0x00, 0x00, 0xf0, 0xf0, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00
+.byte 0x0f, 0x00, 0x00, 0x00, 0x00, 0x0f, 0x00, 0x00, 0x00, 0x00, 0xf0
+closeThread
```
Comment (SethBarberee) [inline data/system_sbin.s:2823]: I would be shocked if there were scripting commands in the System sbin.

File: data/data_8115F5C.s [hunk 4]
```diff
@@ -2007,9 +2008,9 @@ gUnknown_8118350: @ 8118350
 @ replacing .incbin "baserom.gba", 0x118350, 0x50
 .byte 0xf6, 0x00, 0xa6, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00
 .4byte gUnknown_8118334
-.byte 0x54, 0x00, 0x01, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00
+setAnimation 1
```
Comment (SethBarberee) [inline data/data_8115F5C.s:2011]: How positive are we that these are scripting commands?

**Review by SethBarberee — COMMENTED** [review]
Cutscene file changes are fine, mainly questions on changes to files that aren't cutscenes. Not sure if there are scripting commands outside of cutscene files

**Comment by LuisBoto** [comment]
> Cutscene file changes are fine, mainly questions on changes to files that aren't cutscenes. Not sure if there are scripting commands outside of cutscene files

You're right - sorry it does look obvious in hindsight 😅 Still getting familiar with the project, so I'll double check the replaces don't bleed into unrelated files next time. For now I've reverted those out!

**Review by SethBarberee — APPROVED** [review]

**Pull request merged**